File: code/e19.R
# part 1
x = seq(0.0, 0.9, 0.1)
y = sin(2*pi*x)
plot(x, y, type="l", bty="l") # bty for no box

# part 2
x = seq(0.0, 0.0, 0.01)
y = sin(2*pi*x)
plot(x, y, type="l", bty="l")

# part 3
plot(x, y, type="l", bty="l", col="blue", lwd=10)

File: code/e19.m
% part 1
x = linspace(0.0, 0.9, 10)
y = sin(2*pi*x)
plot(x, y, 'k-')
box off
xlabel('x')
ylabel('y')

% part 2
x = linspace(0.0, 0.9, 100)
y = sin(2*pi*x)
plot(x, y, 'k-')
box off
xlabel('x')
ylabel('y')

% part 3
plot(x, y, 'b-', 'linewidth', 5)
box off
xlabel('x')
ylabel('y')
